infra/charts/argo.workflows.ts

```diff
@@ -121,6 +121,11 @@ export class ArgoWorkflows extends Chart {
         },
         artifactRepository,
         controller: {
+          /* Tells Fluent Bit to not send Argo Controller log to CloudWatch
+            https://github.com/argoproj/argo-workflows/issues/11657 is spamming the logs
+            and increase our logs storage cost.
+          */
```

Comment:
```suggestion
          /*
           * Tells Fluent Bit to not send Argo Controller log to CloudWatch
           * https://github.com/argoproj/argo-workflows/issues/11657 is spamming the logs
           * and increase our logs storage cost.
           */
```

not sure if this will format correctly but generally you use a `*` per line.